Common/src/CConfig.cpp

```diff
@@ -3560,6 +3562,17 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
       SU2_MPI::Error(string("You probably want to set INC_ENERGY_EQUATION= YES for the fluid solver. \n"), CURRENT_FUNCTION);
   }
 
+  /*--- Check correctness and consistency of contact resistance options. ---*/
+  if (nMarker_ContactResistance > 0) {
+    ContactResistance = true;
```

Comment:
You don't need the bool, you can just use nMarker_ContactResistance > 0, or even better allocate an array with zeros.